src/Core/Content/ImportExport/Processing/Writer/AbstractFileWriter.php

```diff
@@ -34,6 +34,10 @@ public function flush(Config $config, string $targetPath): void
     {
         rewind($this->buffer);
 
+        if (!\is_resource($this->tempFile)) {
+            $this->initTempFile();
```

Comment:
wouldn't it be better to just open the same temp file every time? Because now you also generate a complete new tmp file each time.
```suggestion
            $this->tempFile = fopen($this->tempPath, 'a+b');
```